incubator/elastic-stack/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
Please update with latest `_helpers.tpl` you get from `helm create`.

Reply:
Done.